python/pylibcudf/pylibcudf/__init__.pxd

```diff
@@ -78,4 +79,5 @@ __all__ = [
     "transpose",
     "types",
     "unary",
+    "nvtext",
```

Comment:
Nit: Could you keep list alphabetical as well?

Reply:
Sure thing, will follow up